README.md

```diff
@@ -1,39 +1,65 @@
-# Temporal  
 [![Build status](https://badge.buildkite.com/fc0e676d7bee1a159916af52ebdb541708d4b9f88b8a980f6b.svg)](https://buildkite.com/temporal/temporal-server)
 [![Coverage Status](https://coveralls.io/repos/github/temporalio/temporal/badge.svg?branch=master)](https://coveralls.io/github/temporalio/temporal?branch=master)
 [![Discourse](https://img.shields.io/static/v1?label=Discourse&message=Get%20Help&color=informational)](https://community.temporal.io)
 
-Visit [docs.temporal.io](https://docs.temporal.io) to learn about Temporal.
+# Temporal  
+
+Temporal is a microservice orchestration platform which enables developers to build scalable applications without sacrificing productivity or reliability.
+Temporal server executes units of application logic, Workflows, in a resilient manner that automatically handles intermittent failures, and retries failed operations.
 
-This repo contains the source code of the Temporal server. To implement workflows, activities and worker use [Go SDK](https://github.com/temporalio/sdk-go) or [Java SDK](https://github.com/temporalio/sdk-java).
+The programming model of Temporal exposed via the SDKs (currently for Java and Go) offers developers the simple abstractions of Workflows and Activities that can virtually "resume" execution after a failure from the same line of code.
 
-See Maxim's talk at [Data@Scale Conference](https://atscaleconference.com/videos/cadence-microservice-architecture-beyond-requestreply) for an architectural overview of Temporal.
+Temporal is a mature technology, a fork of Uber's Cadence.
+Temporal is being developed by [Temporal Technologies](https://temporal.io/), a startup by the creators of Cadence.
+
+[![IMAGE ALT TEXT](http://img.youtube.com/vi/f-18XztyN6c/0.jpg)](http://www.youtube.com/watch?v=f-18XztyN6c "Temporal")
```

Comment:
why the icon is green?

Reply:
http://img.youtube.com/vi/f-18XztyN6c/0.jpg

Reply:
I suspect that's what youtube generated as a still image.

Reply:
I guess we can put our own image instead.

Reply:
Added a better one. Not showing up in preview for some reason.

Reply:
for the pic
could you plz setup git lfs?
```zsh
brew install git-lfs
git lfs install
git lfs track "*.png"
```

Reply:
FYI
in the PR the binary can be shown by click this
![image](https://user-images.githubusercontent.com/8762893/96061180-61ba5380-0e47-11eb-8b88-0103896b3c5c.png)